app/views/user/show.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Show users</h1>
\ No newline at end of file
+<section class="row">
+  <div class="col-12">
+  <%= render partial:'partials/user', locals: {user: @user} %>
+  </div>
+  <div class="col-12">
+    <h4>Bio</h4>
+    <p><%= @user.bio %></p>
+  </div>
+  <div class="col-12">
```

Comment:
Please create a method to show the user's 3 most recent posts and use it inside `app/views/users/show.html.erb`.
Please follow this [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png) for the user's show page.